integration/test_dynamic_links.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Integration tests for firebase_admin.auth module."""
+import sys
+
+import pytest
+import requests
+
+
+from firebase_admin import dynamic_links
+
+from tests import testutils
+
+dynamic_links_e2e_url = ''
+try:
+    dynamic_links_e2e_url = testutils.resource('dynamic_links_e2e_url.txt').strip()
+except IOError:
+    sys.stderr.write("""
```

Comment:
Just output a short statement like `skipping dynamic links integration tests`, and provide detailed instructions elsewhere (ideally in CONTRIBUTING.md)